src/Orleans/Runtime/GrainReference.cs

```diff
@@ -15,6 +15,12 @@ public class GrainReference : IAddressable, IEquatable<GrainReference>, ISeriali
         private readonly string genericArguments;
         private readonly GuidId observerId;
 
+        /// <summary>
+        /// Invoke method options specific to this grain reference instance
+        /// </summary>
+        [NonSerialized]
+        private readonly InvokeMethodOptions invokeMethodOptions;
```

Comment:
Is that doc comment a little misleading, since the value is also copied via the copy constructor?
*Should* it be copied by the copy constructor? I think the answer is no, since the reference should be temporary and should never escape, but I'm not entirely certain yet.